ui/src/components/entity/citation/PaperSummary.tsx

```diff
@@ -168,12 +154,52 @@ export default class PaperSummary extends React.PureComponent<Props, State> {
                   </div>
                 </Tooltip>
               ) : null}
+              {!!paper.inboundCitations ? (
+                <Tooltip
+                  placement="bottom-start"
+                  title={
+                    <React.Fragment>
+                      <strong>
+                        {paper.inboundCitations} citation
+                        {paper.inboundCitations !== 1 ? "s" : ""}
+                      </strong>
+                    </React.Fragment>
+                  }
+                >
+                  <div className="paper-summary__metrics__metric">
+                    <InboundCitationIcon width="12" height="12" />
+                    {paper.inboundCitations}
+                  </div>
+                </Tooltip>
+              ) : null}
+              {!!paper.outboundCitations ? (
+                <Tooltip
+                  placement="bottom-start"
+                  title={
+                    <React.Fragment>
+                      <strong>
+                        {paper.outboundCitations} reference
+                        {paper.outboundCitations !== 1 ? "s" : ""}
+                      </strong>
+                    </React.Fragment>
+                  }
+                >
+                  <div className="paper-summary__metrics__metric">
+                    <OutboundCitationIcon width="12" height="12" />
+                    {paper.outboundCitations}
+                  </div>
+                </Tooltip>
+              ) : null}
             </div>
           ) : null}
-          {inLibrary
-            ? <LibraryButton label="In Your Library" onClick={goToLibrary}/>
-            : <LibraryButton label="Save To Library" onClick={this.saveToLibrary}/>
-            }
+          {inLibrary ? (
```

Comment:
Not going to `!!` this one? Technically, it's only needed for `{!!variable && (...)}`,  but maybe consistency here is more important.

Reply:
Will be addressed in a follow-up change.